### ISSUE TITLE
JAWS/NVDA is not announcing the real time content. It is reading some extra content along with the generated response:A11y_Visual Studio Code_Github_Copilot_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/vscode#260484]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yeDAD; #A11yMAS; #A11yTCS; #Win32; #A11ySev3;#DesktopApp; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #FTP; #Screen Reader; #JAWS; #Element:chatboard; #STP; #NVDA;

## Environment Details:
Application: VS Code
Visual studio code version: 1.103.
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
JAWS version: 2025
NVDA: 2024.4.2

## Repro Steps

1. Turn on Jaws/NVDA screen reader
2. Open Visual Studio Code 
3. Ensure GitHub Copilot is installed and signed in.
4. Open a Python file (e.g., app.py) or create a new one.
5. In the editor, provide a prompt that requires Copilot to generate a large or complex block of code. For example below prompt and observe the issue

"Generate a complete backend server in Python that includes:
- User authentication with JWT
- SQLite database integration
- RESTful endpoints for user CRUD operations
- Input validation
- Logging
- Error handling with appropriate HTTP status codes
- Configuration for CORS and environment variables"

## Actual Experience.
When Copilot is generating the code, screen reader is not announcing the real time content. It is reading some extra content along with the generated response.
Note: Issue is repro with NVDA

## Expected Experience
Screen reader should announce the real time content and it should not read extra content along with the generated response.

## User Impact:
Screen reader user face difficulty if screen reader will not announce the real time response and reading some extra content along with the generated response.

## Attachment
JAWS:

https://github.com/user-attachments/assets/6ead7c70-0140-4c30-bb01-17ff0ec59524

NVDA:

https://github.com/user-attachments/assets/ce843d8d-0657-4635-96bd-335705d0d7c5
